src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -165,6 +165,7 @@
     <PlatformManifestFileEntry Include="API-MS-Win-core-xstate-l2-1-0.dll" IsNative="true" FallbackFileVersion="$(WindowsForwarderFileVersion)" />
     <PlatformManifestFileEntry Include="ucrtbase.dll" IsNative="true" />
     <PlatformManifestFileEntry Include="msquic.dll" IsNative="true" FallbackFileVersion="$(MsQuicFileVersion)" />
+    <PlatformManifestFileEntry Include="libmsquic.dylib" IsNative="true" FallbackFileVersion="$(MsQuicFileVersion)" />
```

Comment:
This is telling Microsoft.NETCore.App packages that libmsquic could be redistributed with the runtime. Is that expected?

Reply:
yes. we do that for Windows and with this we would do that only for debug build so we can run tests. 
This file does not seems to have distinction between debug and relate build so it is added once.

Reply:
Hm, but this is saying libmsquic.dylib will get to Microsoft. NETCore.App.Runtime.osx-x64/arm64. @jkoritzinski, do you see any risk there? Also, mildly surprised about all the manual bit copying that needs to happen.

Reply:
If we aren't planning on shipping it, I'd like to condition this item on the debug configuration like so: `Condition="'$(Configuration)' == 'Debug'"`. That will ensure that we don't accidentally ship it.

If we intent to ship it, then having it here unconditioned is okay.

Reply:
updated. This is not intended for shipping (unlike Windows) so adding the condition make sense to prevent accidents.